content-tool/tool/src/java/org/sakaiproject/content/tool/ResourcesAction.java

```diff
@@ -4310,7 +4310,22 @@ public int compare(Group g0, Group g1) {
 			ContentCollection collection = ContentHostingService.getCollection(collectionId);
 			
 			ListItem item = ListItem.getListItem(collection, null, registry, need_to_expand_all, expandedCollections, items_to_be_moved, items_to_be_copied, 0, userSelectedSort, false, null);
-			
+
+			if(dropboxMode && item.getMembers()!= null && !item.getMembers().isEmpty()) {
+					for (ListItem member:item.getMembers()) {
+						String memberIdArray[] = member.getId().split("/");
+						if(memberIdArray.length>1 && member.isCollection()) {
+							try {
+								String uniqueId = UserDirectoryService.getUser(memberIdArray[3]).getDisplayId();
```

Comment:
Is there a possible `ArrayOutOfBoundsException` here when `memberIdArray` has a length of 2?